src/ChannelServer/World/Character.cs

```diff
@@ -321,11 +321,27 @@ public void LevelUp()
 			//conn.Send(packet);
 		}
 
-		/// <summary>
-		/// Grants exp to character and handles level ups.
-		/// </summary>
-		/// <param name="exp"></param>
-		public void GiveExp(int exp, int jobExp, Monster monster)
+        /// <summary>
+        /// Increases character's level by the given amount of levels.
+        /// </summary>
+        /// <param name="amount"></param>
+        public void LevelUp(int amount)
```

Comment:
As this method basically does exactly the same as the other one, you should combine them, to reduce redundancy. For example, the one without parameter could call this one with amount = 1, or you could remove the other one and give amount a default value of 1.